### DISCUSSION TITLE
Latest Azuracast Rolling release does not have Sync Tasks

### DISCUSSION THREAD

**Discussion by farhad-motechwa** [AzuraCast/AzuraCast#4998] [Q&A]
Has anyone realised that the Sync Tasks are missing from the latest Azuracasr=t Stable version v0.15.0? 

Also the new update has affected the program schedule. Does Any help, please?

**Comment by ItsMitchh** [comment]
If the sync tasks are broken, you can run them from the system debug page and see the error it outputs.

**Comment by BusterNeece** [comment]
We found an issue in the latest stable release for Ansible, and it's fixed in the Rolling Release update for Ansible. We'll be pushing a stable version update shortly once we've verified the Ansible issues.

**Comment by farhad-motechwa** [comment]
Thank you awaiting the update



*Farhad Omar*
*MOTECHWA*
*Putting Technology in Your Hands*
www.motechwa.com.au



*Google Apps System Integrator*

The information contained in this communication is intended solely for the
use of the individual or entity to whom it is addressed and may contain
confidential or legally privileged information. If you are not the intended
recipient, you should not disseminate, distribute, copy or take any action
in reliance on the contents of this communication. If you have received
this communication in error, please notify us immediately by return email
and delete this email from your system.

Email transmission cannot be guaranteed to be secure or error-free. The
sender therefore does not accept liability for any errors or omissions in
the contents of this message which arise as a result of email transmission.
If verification is required, please request a hard-copy version. M O
Technology WA is neither liable for the proper and complete transmission of
the information contained in this communication nor for any delay in its
receipt.

**Comment by farhad-motechwa** [comment]
Need a heads up when the update has been made. 
Cheers!

**Comment by farhad-motechwa** [comment]
Any updates?

**Reply by BusterNeece** [reply]
Like we said, it's been fixed in Rolling Release, and it will be in the next Stable release once we've fixed some other bugs that have been reported. Please don't just post over and over asking for updates; if you want to subscribe to releases, GitHub offers a way for you to do that in this repository to automatically be notified when new releases come out by clicking the "watch" icon in the top right.

**Reply by farhad-motechwa** [reply]
Thanks. Do apologise
